MyCryptoHQ/MyCrypto
# Remove assumption about property names in Dropdown's options

Closes #2042 

### Description

The Dropdown component is given an array of options as a prop. If these option objects don't have a `value` property, the currently selected option isn't being passed as a prop to Select, which results in the selected option not being displayed (though functionally, it still works).

### Changes

This PR removes the assumption about property names by passing in the entire `selectedOption` as Select's initial value. The react-select documentation isn't very explicit about this, but it handles the case where the initial value is an object instead of a string or number.

### Steps to Test

1. Go to Contracts tab
2. Select any contract and choose any function
3. The function name should be displayed in the Read/Write Contract dropdown menu
